merriam-webster.com

piquark6046 opened the pull request:
## Prerequisites

### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

## What problem does the pull request fix?

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [X] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [X] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

## What issue is being fixed?

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/135251

### Add your comment and screenshots

 - https://www.similarweb.com/website/merriam-webster.com/#traffic

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

piquark6046 committed:
FIx #135251 merriam-webster.com leftover

piquark6046 committed:
FIx #135251 merriam-webster.com Social

piquark6046 committed:
Merge branch 'master' into 2022/11/merriamwebstercom-20221123

Alex-302 commented on BaseFilter/sections/specific.txt:
```diff
@@ -275,6 +275,7 @@ chortle.co.uk,coloring.ws,dltk-holidays.com,dltk-kids.com,kidzone.ws,pcsteps.com
 !
 !
 !
+merriam-webster.com##div[id^='mw-ad-']
```
Please always check with AdGuard app. Your rule and Easylists are not applied because of `display: flex!important;` style.
You can make selector of original rule(you can find that rule in AG Extension's filtering log) more specific, and the rule will work:
`merriam-webster.com##.abl`
to
`merriam-webster.com##body .abl`

piquark6046 committed:
Replace with an universal rule

The previous rule works on my end.
However, on some environments, it does not work.

Co-authored-by: Alex-302 <alex302@adguard.com>

piquark6046 committed:
Merge branch 'master' into 2022/11/merriamwebstercom-20221123

piquark6046 committed:
Merge branch 'master' into 2022/11/merriamwebstercom-20221123
